### PR TITLE
About refactor

### LINKED ISSUE #398: pretty sure president should take precedence over event coordinator here
### What happened?

image

### LINKED ISSUE #443: About Me "Year Button" is going through the header
### What happened?

acmpageissue

The button is going through the header page

### PR DESCRIPTION
**Pull request by ghost**
- Refactored what-is-acm component, turned it into about-us component
- Simplified officer-profile-list structure and CSS
- Simplified CSS for officer-profile
- Refactored ACM Select button, CSS now matches ACM Button visually
- Updated board images
- Updated officers.json to reflect proper tiers and board members

### CODE REVIEW

**Comment by jaasonw** [comment]
image
do we need to rebrand this too xd

**Comment by jaasonw** [comment]
also, while it's true that png is lossless and preserves all details, i don't think it's necessary for the officer thumbnails and makes the images load slow (refer to lighthouse report below). would suggest either using jpeg or the fancy new high tech "webp" format

image
https://pagespeed.web.dev/report?url=https%3A%2F%2Facm-csuf-site-git-about-refactor-ethanthatonekid.vercel.app%2Fabout&form_factor=desktop

**Comment by ghost** [comment]
> image do we need to rebrand this too xd

OH FUGGG brb

**Comment by ghost** [comment]
> also, while it's true that png is lossless and preserves all details, i don't think it's necessary for the officer thumbnails and makes the images load slow (refer to lighthouse report below). would suggest either using jpeg or the fancy new high tech "webp" format
> 
> image https://pagespeed.web.dev/report?url=https%3A%2F%2Facm-csuf-site-git-about-refactor-ethanthatonekid.vercel.app%2Fabout&form_factor=desktop

yes yes we will convert to webp

EDIT: performance now at 85 compared to 71

NEW: https://pagespeed.web.dev/report?url=https%3A%2F%2Facm-csuf-site-elw4zd308-ethanthatonekid.vercel.app%2Fabout&form_factor=desktop

OLD: https://pagespeed.web.dev/report?url=https%3A%2F%2Facm-csuf-site-hzmr4kv59-ethanthatonekid.vercel.app%2Fabout&form_factor=desktop

File: src/lib/constants/tiers.json [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
   "Create Officer",
   "Create Project Developer",
-  "Create Event Coordinator",
+  "Create Operations Manager",
   "Dev President",
   "Dev Officer",
   "Dev Project Manager",
-  "NodeBuds Officer",
   "Deprecated Position"
 ]
```
Comment (EthanThatOneKid) [inline src/lib/constants/tiers.json:21]: Any changes made to `src/lib/constants/tiers.json` must be reflected in the form below:

<https://github.com/EthanThatOneKid/acmcsuf.com/blob/74945d6038daba292cacdec6156021c0f1565615/.github/ISSUE_TEMPLATE/officer_update_request.yaml#L44-L72>

File: src/routes/about/index.svelte [hunk 2]
```diff
@@ -1,8 +1,7 @@
 <script lang="ts">
-  import CommonHero from '$lib/components/sections/common-hero.svelte';
-  import OfficerProfileList from '$lib/components/about/officer-profile-list.svelte';
   import Spacing from '$lib/components/sections/spacing.svelte';
-  import Header from '$lib/components/about/what-is-acm.svelte';
+  import About from '$lib/components/about/about-us.svelte';
```
Comment (EthanThatOneKid) [inline src/routes/about/index.svelte]: Nit: Call this component `AboutUs` as to not confuse with the `About` page component.
```suggestion
  import AboutUs from '$lib/components/about/about-us.svelte';
```

**Review by jaasonw — APPROVED** [review]
can confirm it looks like it loads pretty fast now

File: src/lib/components/about/officer-profile.svelte [hunk 3]
```diff
@@ -4,21 +4,17 @@
...
-    .replace(/Dev\s/, `<span class="brand-em brand-bluer">Dev </span>`)
-    .replace(
-      /NodeBuds/,
-      `<span class="headers">node<span class="brand-em brand-red">Buds</span></span>`
-    );
-
+    .replace(/Algo\s/, `<span class="brand-bold acm-purple">Algo&nbsp</span>`)
+    .replace(/Create\s/, `<span class="brand-bold acm-pink">Create&nbsp</span>`)
+    .replace(/Dev\s/, `<span class="brand-bold acm-bluer">Dev&nbsp</span>`)
+    .replace(/nodebuds\s/, `<span class="brand-header">Node Buds&nbsp</span>`);
```
Comment (EthanThatOneKid) [inline src/lib/components/about/officer-profile.svelte]: Nit: End your HTML entities with a semicolon.
```suggestion
    .replace(/Algo\s/, `<span class="brand-bold acm-purple">Algo&nbsp;</span>`)
    .replace(/Create\s/, `<span class="brand-bold acm-pink">Create&nbsp;</span>`)
    .replace(/Dev\s/, `<span class="brand-bold acm-bluer">Dev&nbsp;</span>`)
    .replace(/nodebuds\s/, `<span class="brand-header">Node Buds&nbsp;</span>`);
```

**Review by EthanThatOneKid — CHANGES_REQUESTED** [review]
> can confirm it looks like it loads pretty fast now

Can +1 on @jaasonw's [review](https://github.com/EthanThatOneKid/acmcsuf.com/pull/442#pullrequestreview-946213056). Well done!

I left you a couple nits. Once those are resolved, this PR will be good to go on my end.

File: src/lib/components/about/officer-profile.svelte (hunk 3 above)
Comment (ghost) [reply src/lib/components/about/officer-profile.svelte]: Fixed in commit 8a4f48a

File: src/routes/about/index.svelte (hunk 2 above)
Comment (ghost) [reply src/routes/about/index.svelte]: Fixed in commit 8a4f48a

File: src/lib/constants/tiers.json (hunk 1 above)
Comment (ghost) [reply src/lib/constants/tiers.json:21]: Fixed in commit 8a4f48a

**Comment by EthanThatOneKid** [comment]
This PR tangentially fixes #443.

**Comment by jaasonw** [comment]
O also this fixes https://github.com/EthanThatOneKid/acmcsuf.com/issues/398

**Review by EthanThatOneKid — APPROVED** [review]

**Pull request merged**